src/__test__/editTask.test.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-undef */
+import { editList, verifyEdits, inicializeLists } from "../__mocks__/editTasks";
```

Comment:
- [ ] Kindly rename the file and enable `eslint` as suggested in the previous review.  https://github.com/Aduda-Boaz/ToDoList/pull/11#discussion_r794912216

Reply:
Hello, 
Thank you for pointing out the mistake I was making.
I have made the changes.